interactive_engine/executor/ir/proto/hqps.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package hqps;
+option java_package = "com.alibaba.graphscope.gaia.proto";
+option java_outer_classname = "Hqps";
+
+import "common.proto";
+import "results.proto";
+
+message Argument {
+  string param_name = 1;   // param name
+  int32 param_ind = 2;     // index of param
+  common.Value value = 3;  // real value
+}
+
+message HighQPSQuery {
+  common.NameOrId query_name = 1;
+  repeated Argument arguments = 2;
```

Comment:
Donot call them HighQPS xx. These are StoredProcedureQuery, StoredProcedureResults and StoredProcedureArgument. The name is also `stored_procedure.proto`

Reply:
done